source/tests/yats-test/options_test.cpp

```diff
@@ -0,0 +1,163 @@
+#include <gmock/gmock.h>
+
+#include <yats/options.h>
+#include <yats/scheduler.h>
+#include <yats/slot.h>
+
+using namespace yats;
+
+struct task
+{
+    static options_map<task> options()
+    {
+        return
+        {
+            { "value", &task::val }
+        };
+    }
+
+    output_bundle<slot<int, 0>> run(slot<int, 0> input)
```

Comment:
do we really need the `output_bundle` if we return only one value? I have tests where this is not needed.